receiver/hostmetricsreceiver/internal/scraper/memoryscraper/memory_scraper_others.go

```diff
@@ -22,11 +22,10 @@ import (
 	"go.opentelemetry.io/collector/consumer/pdata"
 )
 
-const availableStateLabelValue = "available"
-
-const memStatesLen = 2
+const memStatesLen = 3
 
 func appendMemoryUsageStateDataPoints(idps pdata.Int64DataPointSlice, memInfo *mem.VirtualMemoryStat) {
 	initializeMemoryUsageDataPoint(idps.At(0), usedStateLabelValue, int64(memInfo.Used))
-	initializeMemoryUsageDataPoint(idps.At(1), availableStateLabelValue, int64(memInfo.Available))
+	initializeMemoryUsageDataPoint(idps.At(1), freeStateLabelValue, int64(memInfo.Free))
+	initializeMemoryUsageDataPoint(idps.At(2), inactiveStateLabelValue, int64(memInfo.Inactive))
```

Comment:
Not sure I understand the difference between inactive and free.

Reply:
Inactive memory is similar to cached in that it can be freed if needed, but it's not counted as part of free.

https://wiki.freebsd.org/Memory
https://superuser.com/questions/590962/memory-usage-classification-under-darwin-mac-os-x-10-8

Not sure this covers all OSs perfectly, but seems to match what we get back from gopsutil for at least Darwin and OpenBSD